packages/runtime-common/realm.ts

```diff
@@ -276,12 +283,12 @@ export class Realm {
       return new Response(null, {
```

Comment:
this is an example of a response that should not be instantiate. There should be no new Response, rather the response that is used should be the one that is passed in.